### PR TITLE
fix: playground config, subgraphs and members table, graph visualization

### PR DESCRIPTION
**Pull request by thisisnithin**
## Motivation and Context

- Select config in playground to query against subgraphs
- Paginate and search for subgraphs and members table
- improve graph visualization in overview
- Preserve schema type in navigation
- Other minor UI fixes

https://github.com/wundergraph/cosmo/assets/44922285/0372e30d-0c1f-4fcd-b44b-416e090da01e

Fixes ENG-5079
Fixes ENG-5193

## TODO

- [ ] Tests or benchmark included
- [ ] Documentation is changed or added on [https://app.gitbook.com/](https://app.gitbook.com/)
- [x] PR title must follow [conventional-commit-standard](https://github.com/wundergraph/wundergraph/blob/main/CONTRIBUTING.md#conventional-commit-standard)

### CODE REVIEW

File: studio/src/components/graph-visualization.tsx [hunk 1]
```diff
@@ -107,6 +116,35 @@ function GraphVisualization({
...
+  const [showAll, setShowAll] = useState(false);
+  const [topCategory, setTopCategory] = useState("latency");
+
+  const subgraphs = useMemo(() => {
+    if (showAll) {
+      return graphData?.subgraphs ?? [];
+    }
+
+    return (
+      graphData?.subgraphs.slice(0, 5).sort((a, b) => {
```
Comment (StarpTech) [inline studio/src/components/graph-visualization.tsx:128]: Why do we need to calculate metrics in the frontend?

**Review by StarpTech — CHANGES_REQUESTED** [review]

File: studio/src/components/graph-visualization.tsx (hunk 1 above)
Comment (StarpTech) [reply studio/src/components/graph-visualization.tsx:128]: Got it. You only sort it.

Comment (thisisnithin) [reply studio/src/components/graph-visualization.tsx:128]: We already have all the data. Local sort is fast. Server side sort will be overkill for this.

**Comment by thisisnithin** [comment]
Filters are also always on in the graph overview for both tabs

Screenshot 2024-05-22 at 4 07 42 PM

https://github.com/wundergraph/cosmo/assets/44922285/43d1bf73-18f1-4c63-b9c7-a636cc492bbe

**Review by jensneuse — APPROVED** [review]

**Pull request merged**